task/__init__.py

```diff
@@ -334,12 +334,13 @@ def push_branch(branch, force=False):
     execute(*cmd)
 
 
-def branch(context, message, pathspec=".", issue=None, push=True, **kwargs):
+def branch(context, message, pathspec=".", issue=None, push=True, branch=None, **kwargs):
     name = named(kwargs)
 
-    branch = f'{name}-{context or ""}-{datetime.utcnow():%Y%m%d-%H%M%M}'
```

Comment:
Can you fix` %M%M` thing while you're at it?  (separate commit, or followup)

Reply:
https://github.com/cockpit-project/bots/pull/3466